step-functions/nino_issue_credential.asl.json

```diff
@@ -153,7 +130,16 @@
         "credentialSubject": {
           "name": [
             {
-              "nameParts.$": "$.map"
+              "nameparts": [
+                {
+                  "type.$": "States.ArrayGetItem($.userDetails.Items[0].names.L[*].M.nameParts.L[?(@.M.type.S==GivenName)][*].type.S, 0)",
```

Comment:
What does the output of this look like?

Reply:
Check your state machine -> `suraj-pdv-matching-NinoIssueCredential`

Reply:
The previous execution isn't there anymore, I'll redeploy these changes and have a test